src/components/Cart/Cart.js

```diff
@@ -66,17 +57,17 @@ const Cart = (props) => {
             {cartItems?.length > 0 &&
               cartItems?.map((item, i) => {
                 return (
-                  <div key={i + "cartItem"} className="cart-items">
+                  <div key={`${i}-cartItem`} className="cart-items">
                     <div>
                       <input
                         type="checkbox"
                         checked={selectedItems.includes(i)}
                         onChange={() => handleCheckboxChange(i)}
                       />
-                      <b>Title:</b> {item?.title}
```

Comment:
this is a rendering check so that if the item isn't fully loaded it will not break the site please leave the question marks in front of the periods... thank you :)